README.md

```diff
@@ -6,27 +6,18 @@
 
 <h1 align="center">Sylius Standard Edition</h1>
 
-<p align="center">This is Sylius Standard Edition repository for starting new projects.</p>
+<p align="center">This is Sylius Customization repository with all code uses in customization guides</p>
 
-About
+Customization Guide
 -----
 
-Sylius is the first decoupled eCommerce framework based on [**Symfony**](http://symfony.com) and [**Doctrine**](http://doctrine-project.org). 
-The highest quality of code, strong testing culture, built-in Agile (BDD) workflow and exceptional flexibility make it the best solution for application tailored to your business requirements. 
-Enjoy being an eCommerce Developer again!
-
-Powerful REST API allows for easy integrations and creating unique customer experience on any device.
-
-We're using full-stack Behavior-Driven-Development, with [phpspec](http://phpspec.net) and [Behat](http://behat.org)
-
-Documentation
--------------
-
-Documentation is available at [docs.sylius.org](http://docs.sylius.org).
+All customization guides are available at [**This address**](https://docs.sylius.com/en/1.4/customization/index.html).
 
 Installation
```

Comment:
Everything below is not needed

Reply:
Okay, let's keep the License and Authors.

And before let's add links to Sylius/Sylius and Sylius/Sylius-Standard